docs/cep/query-guide/functions/source/kafka.md

```diff
@@ -8,25 +8,24 @@ with a partition for a Kafka cluster. The events received can be in the
 | enable.offsets.commit  | This parameter specifies whether to commit offsets. If the manual asynchronous offset committing is needed, `enable.offsets.commit` should be `true` and `enable.auto.commit` should be `false`. If periodical committing is needed `enable.offsets.commit` should be `true` and `enable.auto.commit` should be `true`. If committing is not needed, `enable.offsets.commit` should be `false`. Note: `enable.auto.commit` is an `optional.configuration` property. If it is set to `true`, Source will periodically(default: 1000ms. Configurable with `auto.commit.interval.ms` property as an `optional.configuration`) commit its current offset (defined as the offset of the next message to be read) for the partitions it is reading from back to Kafka. To guarantee at-least-once processing, we recommend you to enable Stream App Periodic State Persistence when `enable.auto.commit` property is set to `true`. During manual committing, it might introduce a latency during consumption. | true          | BOOL                | Yes      | No      |
-| optional.configuration | This parameter contains all the other possible configurations that the consumer is created with. e.g., `ssl.keystore.type:JKS,batch.size:200`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | null          | STRING              | Yes      | No      |
+| optional.configuration | This parameter contains all the other possible configurations that the consumer is created with. e.g., `ssl.keystore.type:JKS,batch.size:200`.                               | null          | STRING              | Yes      | No      |
 
 ## Example 1
```

Comment:
Similar changes like in the sink->kafka, should be made in this file.
@info(name = 'query1) makes more sense to go above the query than above sink creation.
Also, map.type should be json instead of xml, since we don't currently support xml.
This should be changed in both examples.

Reply:
Please either make a commit on this branch PR or make the changes in suggestions. To make a suggestion, click the icon with a plus and minus in the comment editor.

Reply:
@bdzekic1, I do not understand what changes you are asking for. Please either make a commit or make a suggestion as I described.

Reply:
I have, you should be able to see changes right now @diana-macrometa

Reply:
I meant a commit on this PR. It's fine, I'll reconcile the conflicts once the other PR is merged.

In future, please feel free to put a commit on the PR branch (move-apache-beta in this case). Then I won't have to reconcile conflicts later.

Reply:
Understood. Sorry for the misunderstanding. Will keep this in mind for future PRs. There will probably be another PR that I did outside of this branch if you haven't received it already.